modules/pipeline/providers/edgepipeline_register/interface_test.go

```diff
@@ -238,3 +238,42 @@ func TestIsEdge(t *testing.T) {
 	}
 	assert.Equal(t, true, p.IsEdge())
 }
+
+func TestShouldDispatchToEdge(t *testing.T) {
```

Comment:
it seems to be an unrelated unit test.

Reply:
yes, `ShouldDispatchToEdge` only use `IsClusterDialerClientRegistered` to judge is registed